File: primes/src/primes.go
package main

import "fmt"

var prime = 8498810289638043367

func primeFactors() []int {
	factors, divisor, n := make([]int, 0), 2, prime

	for n >= 2 {
		if n%divisor == 0 {
			factors = append(factors, divisor)
			n = n / divisor
		} else {
			divisor++
		}
	}
	return factors
}

func main() {
	fmt.Println(fmt.Sprintf("%s", fmt.Sprint(primeFactors())))
}

File: primes/src/primes.js
const prime = 8498810289638043367n;

function primeFactors(n) {
  const factors = [];
  let divisor = 2n;

  while (n >= 2n) {
    if (n % divisor == 0n) {
      factors.push(divisor);
      n = n / divisor;
    } else {
      divisor++;
    }
  }
  return factors;
}

const factors = primeFactors(prime);

console.log(factors)